--- a/debian/com.raspberrypi.rpi-imager.metainfo.xml.in
+++ b/debian/com.raspberrypi.rpi-imager.metainfo.xml.in
@@ -3,6 +3,9 @@
   <id>com.raspberrypi.rpi-imager</id>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>Apache-2.0</project_license>
+  <developer id="raspberrypi.com">
+    <name>Raspberry Pi Ltd</name>
+  </developer>
   <name>Raspberry Pi Imager</name>
   <summary>Raspberry Pi Imaging utility</summary>
   <description>
@@ -30,23 +33,23 @@
   <launchable type="desktop-id">com.raspberrypi.rpi-imager.desktop</launchable>
   <screenshots>
     <screenshot type="default">
-      <image>http://downloads.raspberrypi.org/imager/IMAGING-UTILITY-MAIN.png</image>
+      <image>https://downloads.raspberrypi.org/imager/IMAGING-UTILITY-MAIN.png</image>
       <caption>Main window</caption>
     </screenshot>
     <screenshot>
-      <image>http://downloads.raspberrypi.org/imager/IMAGING-UTILITY-OS.png</image>
+      <image>https://downloads.raspberrypi.org/imager/IMAGING-UTILITY-OS.png</image>
       <caption>Choose OS</caption>
     </screenshot>
     <screenshot>
-      <image>http://downloads.raspberrypi.org/imager/IMAGING-UTILITY-SD.png</image>
+      <image>https://downloads.raspberrypi.org/imager/IMAGING-UTILITY-SD.png</image>
       <caption>Choose SD</caption>
     </screenshot>
     <screenshot>
-      <image>http://downloads.raspberrypi.org/imager/IMAGING-UTILITY-WRITE.png</image>
+      <image>https://downloads.raspberrypi.org/imager/IMAGING-UTILITY-WRITE.png</image>
       <caption>Write in progress</caption>
     </screenshot>
     <screenshot>
-      <image>http://downloads.raspberrypi.org/imager/IMAGING-UTILITY-DONE.png</image>
+      <image>https://downloads.raspberrypi.org/imager/IMAGING-UTILITY-DONE.png</image>
       <caption>Write done</caption>
     </screenshot>
   </screenshots>
@@ -55,7 +58,7 @@
     <binary>rpi-imager</binary>
   </provides>
   <releases>
-    <release version="@IMAGER_VERSION_STR@" />
+    <release version="@IMAGER_RELEASE_VERSION@" date="@IMAGER_RELEASE_DATE@" />
   </releases>
   <content_rating type="oars-1.1">
     <content_attribute id="social-info">moderate</content_attribute>
